src/Microsoft.Azure.WebJobs.Extensions.Storage/Queues/Config/QueuesOptions.cs

```diff
@@ -34,6 +35,12 @@ public class QueuesOptions : IOptionsFormatter
         public QueuesOptions()
         {
             _newBatchThreshold = -1;
+
+            string skuValue = Environment.GetEnvironmentVariable(Constants.AzureWebsiteSku);
```

Comment:
My only feedback is to put this into a shared helper, e.g. Utility.GetProcessorCountForSku(), since the code is replicated a bunch below (mostly in tests).That way this code and the tests just call the helper

Reply:
Added the utility function as suggested